src/app/profile/yourGiving/editRecurringGifts/step4/confirmRecurringGifts.component.js

```diff
@@ -32,6 +32,7 @@ class ConfirmRecurringGiftsController {
 
   saveChanges () {
     this.analyticsFactory.track('aa-edit-recurring-submit')
+    this.analyticsFactory.trackGTM('ga-edit-recurring-submit')
```

Comment:
A lot of these events don't need any additional data, so I just used the ```trackGTM``` function and changed the event name to align with google analytics.